packages/api/src/routes/medical/document.ts

```diff
@@ -24,34 +31,32 @@ const router = Router();
  * @param req.query.patientId Patient ID for which to list documents.
  * @param req.query.dateFrom Optional start date that docs will be filtered by (inclusive).
  * @param req.query.dateTo Optional end date that docs will be filtered by (inclusive).
+ * @param req.query.organization Optional name of the contained Organization to filter docs
+ *    by (partial match and case insentitive).
+ * @param req.query.content Optional value to search on the document reference
+ *    (partial match and case insentitive, minimum 3 chars).
+ * @param req.query.output Optional value indicating the output format, fhir or dto.
+ *    (default: fhir)
  * @return The available documents, including query status and progress - as applicable.
  */
 router.get(
   "/",
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
     const patientId = getFromQueryOrFail("patientId", req);
-    const dateFrom = parseISODate(getFrom("query").optional("dateFrom", req));
-    const dateTo = parseISODate(getFrom("query").optional("dateTo", req));
+    const { dateFrom, dateTo, content, output } = getDocSchema.parse(req.query);
```

Comment:
would it be useful to sanitize the input content string from special characters, trim it, or limit the input length?

Reply:
yeah, but it requires some thinking about what to remove, as well as document it. For now, I created a sanitize function (that we should have anyways), it simply trims the string. Could look into something like this https://www.npmjs.com/package/string-sanitizer